recipes/giotto-learn/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ name }}/{{ name }}.git
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - git submodule update --init --recursive
+    - cmake -G "Visual Studio 15 2017"  # [win]
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - vs2017_win-64  # [win]
+    - cmake
+    - boost >=1.56.*
+    - git
+    - setuptools
```

Comment:
``boost`` and ``setuptools`` should be in host only

Reply:
done! thank you!